storybook/src/data/documents.data.js

```diff
@@ -0,0 +1,60 @@
+import uuid from 'uuid/v4';
+
+export default class DocumentBuilder {
+  entityType = 'document';
+
+  uid = uuid();
```

Comment:
This should be part of build... otherwise you won't be able to use the same instance to generate several alike objects.